### PR TITLE
add schemas

### PR DESCRIPTION
**Pull request by lukasj**
Signed-off-by: Lukas Jungmann <lukas.jungmann@oracle.com>

### CODE REVIEW

File: jaxb-api/src/main/resources/jakarta/xml/bind/bindingschema_1_0.xsd [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version = "1.0" encoding = "UTF-8"?>
 <!--
 
-    Copyright (c) 2009, 2018 Oracle and/or its affiliates. All rights reserved.
+    Copyright (c) 2009, 2020 Oracle and/or its affiliates. All rights reserved.
```
[the remark this answers is not in the document]
Comment (lukasj) [reply jaxb-api/src/main/resources/jakarta/xml/bind/bindingschema_1_0.xsd]: That would be true for the right source. This file in particular was at wrong location in the repo (move -> cp year change to satisfy copyright plugin) and it was wrong version/revision - take a look at `inlineBinaryData` element - it was present in this version but it is not defined for 1.0, it was added during v. 2.0 preparation while few other elements required by 2.0 are missing. I only fixed this to make sure that v 1.0 is really v 1.0 like [this one](http://www.oracle.com/webfolder/technetwork/jsc/xml/ns/jaxb/bindingschema_1_0.xsd)

File: jaxb-api/src/main/resources/jakarta/xml/bind/bindingschema_3_0.xsd [hunk 2]
```diff
@@ -0,0 +1,460 @@
...
+    terms of the Eclipse Distribution License v. 1.0, which is available at
+    http://www.eclipse.org/org/documents/edl-v10.php.
+
+    SPDX-License-Identifier: BSD-3-Clause
+
+-->
+
+<xs:schema
+   targetNamespace = "http://xmlns.jakarta.ee/xml/ns/jaxb"
+   xmlns:jaxb = "http://xmlns.jakarta.ee/xml/ns/jaxb"
```
[the remark this answers is not in the document]
Comment (lukasj) [reply jaxb-api/src/main/resources/jakarta/xml/bind/bindingschema_3_0.xsd]: done

File: jaxb-api/src/main/resources/jakarta/xml/bind/bindingschema_2_0.xsd [hunk 3]
```diff
@@ -0,0 +1,460 @@
...
+
+-->
+
+<xs:schema
+   targetNamespace = "http://java.sun.com/xml/ns/jaxb"
+   xmlns:jaxb = "http://java.sun.com/xml/ns/jaxb"
+   xmlns:xs = "http://www.w3.org/2001/XMLSchema"
+   elementFormDefault = "qualified"
+   attributeFormDefault = "unqualified">
+  <xs:annotation>
```
[the remark this answers is not in the document]
Comment (lukasj) [reply jaxb-api/src/main/resources/jakarta/xml/bind/bindingschema_2_0.xsd]: added

File: jaxb-api/src/main/resources/jakarta/xml/bind/bindingschema_1_0.xsd (hunk 1 above)
[the remark this answers is not in the document]
Comment (bshannon) [reply jaxb-api/src/main/resources/jakarta/xml/bind/bindingschema_1_0.xsd]: I see.  Looks like the original source wasn't maintained properly.

**Comment by lukasj** [comment]
@bshannon done; the only remaining question is regarding [1 or 2 copyrights](https://github.com/eclipse-ee4j/jax-ws-api/pull/111#discussion_r399251181)

**Comment by lukasj** [comment]
@bshannon should be done too now

**Pull request merged**